src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -60,11 +61,15 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
         public BindingConfiguration CurrentConfiguration { get; private set; }
 
         [ImportingConstructor]
-        public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolutionTracker, ILogger logger)
+        public ActiveSolutionBoundTracker(IHost host,
+            IActiveSolutionTracker activeSolutionTracker,
+            IBoundSolutionGitMonitor gitEvents,
+            ILogger logger)
         {
-            extensionHost = host ?? throw new ArgumentNullException(nameof(host));
```

Comment:
No point in throwing on nulls - MEF won't call the constructor with nulls (unless you are using the `ImportMany` attribute).